sections/conclusion.tex

```diff
@@ -1 +1,34 @@
-\section{Conclusion}\label{sec:conclusion}
\ No newline at end of file
+\section{Conclusion}\label{sec:conclusion}
+We explore possibilities for incorporating metadata into existing topic models, such as \gls{lda} and \gls{pam}.

```

Comment:
```suggestion
In this paper, we explore possibilities for incorporating metadata into existing topic models, such as \gls{lda} and \gls{pam}.

```

Reply:
Det er bare fyld ord vil jeg mene :)

Reply:
jeg syntes det andet er en sær måde at starte en konklusion på, idk :/
@Starrock45 ? hvad siger du

Reply:
Jeg har ikke et stort problem med "in this paper" her. Det gør det muligvis lidt tydeligere at vi snakker om hele artiklen i dette afsnit. Jeg kan også se at nogle andre artikler gør det samme. :)